CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ set(CMAKE_OBJCOPY arm-none-eabi-objcopy)
 set(CMAKE_OBJDUMP arm-none-eabi-objdump)
 set(SIZE arm-none-eabi-size)
 
-SET(LINKER_SCRIPT ${CMAKE_SOURCE_DIR}/STM32F446RETX_FLASH.ld)
+SET(LINKER_SCRIPT ${CMAKE_SOURCE_DIR}/STM32F446RETX_FLASH.ld Lib/Gps/gps.cpp Lib/Gps/Inc/Gps/gps.hpp)
```

Comment:
added files need to be removed. When adding files to a project in clion a popup comes up, make sure to uncheck automatic additions to cmake.